modules/openxr/doc_classes/OpenXRAPIExtension.xml

```diff
@@ -94,13 +106,34 @@
 				Returns [code]true[/code] if OpenXR is enabled.
 			</description>
 		</method>
+		<method name="register_composition_layer_provider">
+			<return type="void" />
+			<param index="0" name="extension" type="OpenXRExtensionWrapperExtension" />
+			<description>
+				Registers the given extension as a composition layer provider.
+			</description>
+		</method>
+		<method name="set_emulate_environment_blend_mode_alpha_blend">
+			<return type="void" />
+			<param index="0" name="enabled" type="bool" />
+			<description>
+				If set to [code]true[/code], an OpenXR extension is loaded which is capable of emulating the [constant XRInterface.XR_ENV_BLEND_MODE_ALPHA_BLEND] blend mode.
+			</description>
+		</method>
```

Comment:
We can do this automatically on our end instead of letting the gdextension dictates whether this should be set or not, by going through the list of gdextensions and checking their return values to `is_environment_blend_mode_alpha_blend_supported()`.

This would mitigate the issue where multiple gdextensions can be loaded each with different level of support. For example, one gdextension may have full support while the other may only be able to emulate support. The latter would set this flag which would cause the logic to assume that emulation is required when it's not.